Rhino.Templates/content/CSRhinoTest/MyRhino__1Project.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	
+  <PropertyGroup>
+    <!-- Select the framework(s) you wish to target.
+        Rhino 8 Windows: net48, net7.0-windows, net7.0-windows10.0.22000.0, etc
+    -->
+    <TargetFrameworks>net7.0-windows;net48</TargetFrameworks>
+    <LangVersion>11</LangVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <Version>1.0</Version>
+    <Title>MyRhino.1</Title>
+    <Company>MyRhino.1 Authors</Company>
+    <Description>Description of MyRhino.1</Description>
+
+    <NoWarn>NU1701;MSB3277</NoWarn>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="5.0.0" />
+    <PackageReference Include="NUnit" Version="3.14.0" />
```

Comment:
Is there a reason why we aren't using the latest NUnit and Microsoft.NET.Test.Sdk here?

Reply:
None in particular, latest and greatest is good with me.